watch-package.js

```diff
@@ -76,8 +78,9 @@ module.exports = function watchPackage (pkgDir, exit) {
     var proc = processes[script] = spawn(nodemon, args, {
       env: process.env,
       cwd: pkgDir,
-      stdio: 'pipe'
+      stdio: inherit === true? 'inherit' : 'pipe'
```

Comment:
Confirmed to work in both MacOS Sierra and ArchLinux (Linux 4.6.3-1-ec2). Both using Bash v4.3